core/state/access_witness.go

```diff
@@ -131,11 +131,8 @@ func (aw *AccessWitness) TouchTxOriginAndComputeGas(originAddr []byte) uint64 {
 }
 
 func (aw *AccessWitness) TouchTxExistingAndComputeGas(targetAddr []byte, sendsValue bool) uint64 {
-	aw.touchAddressAndChargeGas(targetAddr, zeroTreeIndex, utils.BasicDataLeafKey, false)
```

Comment:
actually I'm thinking we should set it to `true` because this isn't charged, it's just covered by the `21000` gas you pay for the tx.

Reply:
ah nvm, it was specified as such.